common/include/scipp/common/span/test/catch.hpp

```diff
@@ -4757,7 +4757,7 @@ namespace Catch {
 
 // start catch_fatal_condition.h
 
-// start catch_windows_h_proxy.h
+// start catch_windows_h_view.h
```

Comment:
`span` is not our code but a subproject (https://github.com/tcbrindle/span), can you undo the changes in that folder?

Reply:
Done. (apparently you can revert files from GitHub Files Changed tab now)